python/fledge/common/configuration_manager.py

```diff
@@ -348,6 +361,20 @@ def get_entry_val(k):
                             raise ValueError('For {} category, listName cannot be empty for item name '
                                              '{}'.format(category_name, item_name))
                         item_val['listName'] = list_name
+                    elif "permission" in item_val:
+                        permission = item_val['permission']
+                        if not isinstance(permission, list):
+                            raise ValueError(
+                                'For {} category, permission entry value must be in list for item name {}; got {}.'
+                                ''.format(category_name, item_name, type(permission)))
+                        if not permission:
+                            raise ValueError(
+                                'For {} category, permission entry value must not be an empty for item name {}.'.format(
+                                    category_name, item_name))
+                        else:
+                            if not all(isinstance(ev, str) and ev != '' for ev in permission):
+                                raise ValueError('For {} category, permission entry values must be in string and '
```

Comment:
For {} category, permissions value should be a list of user roles

in string is not clear